### DISCUSSION TITLE
Error Starting Flow

### DISCUSSION THREAD

**Discussion by MithunKinarullathil** [MOV-AI/movai-flow#201] [Q&A] [answered]
## [Copied from the forum](https://forum.flow.mov.ai/t/error-starting-flow/59)

[LycanOak](https://forum.flow.mov.ai/u/LycanOak)
[Jun 22](https://forum.flow.mov.ai/t/error-starting-flow/59?u=mithunkinarullathil)
I’ve been using Flow for a few days. Today, when I started it, some error occurred when starting the services.

In the image below, I share a screen capture of the terminal where Flow is being started.

![image](https://user-images.githubusercontent.com/51120171/204017062-428a033b-75d8-4e0f-9e8d-355171354c5e.png)

**Comment by MithunKinarullathil** [comment]
[hexarotor](https://forum.flow.mov.ai/u/hexarotor)
Leader
[Jun 22](https://forum.flow.mov.ai/t/error-starting-flow/59/2?u=mithunkinarullathil)
Welcome to the Forum and thanks for the post

Probably some containers were not properly stopped and the system is now on a strange status

Can you please try 1) to stop all containers typing movai-flow-stopand then 2) launch movai flow again typing movai-flow-launch

**Comment by MithunKinarullathil** [comment, marked as answer]
[LycanOak](https://forum.flow.mov.ai/u/LycanOak)
[Jun 22](https://forum.flow.mov.ai/t/error-starting-flow/59/3?u=mithunkinarullathil)
I tried the steps but the error persisted.

I proceeded to restart my PC and everything is now working correctly.
Thank you

**Comment by MithunKinarullathil** [comment]
[MithunKinarullathil](https://forum.flow.mov.ai/u/MithunKinarullathil)
[Jun 23](https://forum.flow.mov.ai/t/error-starting-flow/59/4?u=mithunkinarullathil)
Hi, thank you for the report. If you have the same issue persisting, please feel free to add an issue here: [GitHub - MOV-AI/movai-flow: MOVAI Flow - The IDE that brings visualization and structure to ROS](https://github.com/MOV-AI/movai-flow)

**Comment by marcusvinicius178** [comment]
I am having a similar issue. The I have issued on terminal the following cmds:


`1)movai-flow-stop
`
`2) movai-flow-launch
`
3) Then I had this error :
![image](https://user-images.githubusercontent.com/43826438/232919367-a8400c17-9254-4059-888e-b2d47e9f9c17.png)

4) To solve it I have stopped and removed this container

![image](https://user-images.githubusercontent.com/43826438/232919782-c701c288-7999-4ddf-aa28-5c3e7020678c.png)

5) Then relaunched the application using host network (by issuing movai-flow-launch on terminal)

Unfortunately the application does not open

![image](https://user-images.githubusercontent.com/43826438/232919239-a67d9f28-0d94-4397-97de-8acd9e2e3a2f.png)

6) I have rebooted and the same errors happened from 1 to 5 again.
